app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationConfirmationView.scala.html

```diff
@@ -42,8 +43,12 @@ <h2 id="declarationDateId" class="govuk-heading-m" id="main-message">@messages("
  <h2 id="whatToDoNextId" class="govuk-heading-m" id="main-message">@messages("declarationConfirmation.h2.1")</h2>
  <p class="govuk-body">@messages("declarationConfirmation.ul.p")</p>
  <ul id="whatToDoNextUlId" class="govuk-list govuk-list--bullet">
-  <li>@messages("declarationConfirmation.ul.1")</li>
+  @if(declaration.declarationType == Export) { @Html("")
```

Comment:
Would it not make sense to have if == Import, then no need for empty Html("") and else statement